app/src/main/res/drawable/ic_hedvig_wordmark_black.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:width="94dp" android:height="30dp" android:viewportWidth="94" android:viewportHeight="30">
```

Comment:
Do we have an existing wordmark we can tint here?

If not, make sure to run `avocado` on this VectorDrawable 🙂